tests/test_search_aggregates.py

```diff
@@ -112,6 +112,7 @@ def generate_addon(addon_id, name, version):
     ("channel", "release", StringType(), True),
     ("country", "DE", StringType(), True),
     ("locale", "de", StringType(), True),
+    ("user_pref_browser_search_region", "DE", StringType(), False),
```

Comment:
Should probably test this being null. You should be able to do that below around line 347